node/clients/mail.ts

```diff
@@ -38,7 +38,7 @@ export class MailClient extends JanusClient {
     return this.http.post(TEMPLATE_RENDER_PATH, template, {
       headers: {
         ...this.options?.headers,
-        VtexIdClientAutCookie: this.context.adminUserAuthToken,
+        VtexIdClientAutCookie: this.context.adminUserAuthToken ?? '',
```

Comment:
Why are we passing this as an empty string?

Reply:
Simple validation, maybe it's redundant, but it's a good practice to have it. But it is considering always the `adminUserAuthToken` and only if the `adminUserAuthToken` is `null` or `undefined` it will get an empty string.

Reply:
@edyespinal if you prefer, we can remove it, although it doesn't affect the feature itself.

Reply:
@lucasayb, again, sorry I didn't make myself clear 😅
I understand the purpose of the code, I was just wondering if you were getting any errors or having any issues when publishing email templates.